src/main/java/campus/tasks/Task.java

```diff
@@ -0,0 +1,43 @@
+package campus.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class object which represents a Task, which has a name and a status completion
+ */
+abstract public class Task {
+    public boolean completed;
+    public String taskName;
+    public DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HHmm dd/MM/yyyy");
+
+    /**
+     * Checks to see if the input string is of the correct format pattern
+     * @param input timedate string
+     * @param formatter HHmm dd/MM/yyyy
+     * @return true if yes, false if no
+     */
+    public static boolean isValidDateTimeFormat(String input, DateTimeFormatter formatter) {
+        try {
+            LocalDateTime.parse(input, formatter);
+            return true;
+        } catch (DateTimeParseException e) {
+            return false;
+        }
+    }
+
+    public static void main(String[] args) {
+
+    }
+
+    public void markComplete() {
```

Comment:
It would be better to mark the method as abstract so that the subclass does not forget to override it